sdks/java/extensions/euphoria/euphoria-beam/src/test/java/org/apache/beam/sdk/extensions/euphoria/beam/OperatorsTestSuite.java

```diff
@@ -15,23 +15,48 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.sdk.extensions.euphoria.operator.test;
+package org.apache.beam.sdk.extensions.euphoria.beam;
```

Comment:
This package looks a little odd to me. There is twice `beam`, could we find a little more descriptive subpackage name? Or maybe just drop the last `beam`?

Reply:
I agree, the package name deserves better name. There is more cleanup work we need to do. So I created new [BEAM-4428](https://issues.apache.org/jira/browse/BEAM-4428) to serve as cleanup task TODO list.

Reply:
👍